### DISCUSSION TITLE
changeDadCharacter function in modcharts.

### DISCUSSION THREAD

**Discussion by AmberHaxe** [KadeArchive/Kade-Engine#1094] [Q&A]
How do you use it? everytime I try and change the string to another character I get a nil value,  am I doing something wrong?

**Comment by AmberHaxe** [comment]
nvm lol figured it out

**Reply by Ibexlol** [reply]
how do you do it?

**Reply by AmberHaxe** [reply]
put it in stepHit instead of update, and make sure you have an if curStep == insert number instance,

then just use changeDadCharacter('insertname'), and whala!

although be warned, it does give a 0.3 second lag spike, thats coming from someone with a relatively good rig.

so in all,

function stepHit (step)
  if curstep == number then
       changeDadCharacter('character')
end

hope that helped!
